typing/index.ts

```diff
@@ -1,5 +1,7 @@
 export type BytesLike = HexString | Uint8Array
 export type BigNumberish = string | number | bigint
 export type HexString = string
+export type UrlB64String = string
+export type UriString = string
```

Comment:
nit: Maybe call it `UrlString` to align with `UrlB64String`?

Reply:
I'd like to revert it back to `string` type here.
Initially, the intention was to encode the stored user value using URI encoding, but later in the code, URI encode/decode processes have been implicitly handled within various functions:
uri encode: [background/messages/mCreateWindow.ts#L31](https://github.com/pilagod/waallet-extension/blob/Implement_passkeyOwnerWebauthn/background/messages/mCreateWindow.ts#L31)
uri decode: [packages/webAuthn/index.ts#L38](https://github.com/pilagod/waallet-extension/blob/Implement_passkeyOwnerWebauthn/packages/webAuthn/index.ts#L38)